src/primitives.jl

```diff
@@ -179,6 +181,25 @@ end
 dropnull(X::NullableVector) = X.values[!X.isnull]    # -> Vector
 
 """
+    dropnull!(X::AbstractVector)
+
+Remove null entries of `X` in-place and return the updated vector. No-op
+if no nulls are present. If `X` is a `NullableVector`, a view of the
+unwrapped values is returned for convenience, although `X` itself will still
```

Comment:
I wouldn't say "although X itself..." as this is a fundamental rule of the language. OTOH, I would move the docs for `NullableVector` to a separate docstring, and I would specify "a `Vector` view".